docs/_api/compositecontroller.md

```diff
@@ -368,3 +369,7 @@ response because they're in different forms.
 Instead, you should think of each entry in the list of `children` as being
 sent to [`kubectl apply`][kubectl apply].
 That is, you should [set only the fields that you care about](/api/apply/).
+
+Note that your webhook handler must return a response with a status code of `200`
+to be consider successful. Metacontroller will wait for a response for up to the
+amount defined in `timeout`, which defaults to `10s` if not defined explicitly.
```

Comment:
Since this is under the "Sync Hook Response" heading, readers might be confused where exactly the `timeout` field lives. To make it clear, and to consolidate the info in only one place, I suggest linking to the general Webhook docs like:

```md
... for up to the timeout defined in the [Webhook spec](/api/hook/#webhook).
```

The reader can click through to find the default value, so we don't have to update it everywhere if it changes.

Reply:
Fixed